project-apps-dev/ProgrammersOnly/mongodb_service.tf

```diff
@@ -41,7 +46,10 @@ resource "aws_ecr_repository_policy" "mongodb" {
                 "ecr:DeleteRepository",
                 "ecr:BatchDeleteImage",
                 "ecr:SetRepositoryPolicy",
-                "ecr:DeleteRepositoryPolicy"
+                "ecr:DeleteRepositoryPolicy",
+                "s3:GetObject",
+                "s3:GetBucketLocation",
+                "kms:*"
```

Comment:
What is the purpose of adding it here? 
This is policy applied only to repository. I guess this should be added to `ECS_EC2` policy `ec2_instance_profile.tf`

Reply:
Added.